crowbar_framework/app/models/api/upgrade.rb

```diff
@@ -278,11 +278,19 @@ def maintenance_updates_check_errors(check)
         if check[:repositories_missing]
           ret[:repositories_missing] = {
             data: I18n.t("api.upgrade.prechecks.repos_missing.error",
-              missing: check[:repositories_missing].join(", ")),
+              missing: check[:repositories_missing]),
```

Comment:
But on 'the other end', the join is used also when doing adminrepocheck, so having join there seems to be consistent when analyzing the output of check_repositories method